README.md

```diff
@@ -655,7 +655,7 @@ nt-dates/data/](https://amiibereval2018.wordpress.com/important-dates/data/)
 
 #### The Hateful Memes Challenge: Detecting Hate Speech in Multimodal Memes
 * Link to publication: [https://arxiv.org/pdf/2005.04790.pdf](https://arxiv.org/pdf/2005.04790.pdf)
-* Link to data: [https://www.drivendata.org/competitions/64/hateful-memes/](https://www.drivendata.org/competitions/64/hateful-memes/)
+* Link to data: [https://hatefulmemeschallenge.com/](https://hatefulmemeschallenge.com/) Get complete data after filling out the form.  
```

Comment:
I get an https security warning for this URL, do you get one too?

Reply:
yes, but we can still visit it with advanced settings.

Reply:
ok, let's move forward once that's fixed and the https works properly